### ISSUE TITLE
ENH: support databroker export

### ISSUE THREAD

**Issue by CJ-Wright** [xpdAcq/xpdAcq#326]
Now that the databroker has export capabilities we should try to support it in such a way that we can take the entire databroker home with all of our metadata and such.

**Comment by chiahaoliu** [comment]
For this functionality, should we make it optional? since there might not be a huge number of users who have all the requirements installed.

From xpdacq side, maybe we can help define search kargs for current beamtime. Although I feel user who is doing export would already be able to set this.

**Comment by CJ-Wright** [comment]
Optional is fine, although we don't have any requirements on the users side to save their databroker on a disk (reading it is another thing). My hope is that we can provide helper functions to store/get all the headers created in the beamtime instance and pass them db.export(). We might even be so kind as to prep a portable-mds/portable-fs instance in the location of their choosing (potentially a USB drive).

**Comment by chiahaoliu** [comment]
yeah, all sounds good. I think we are expecting a thin wrapper for db.export. (pass desired/all headers to export function)

**Comment by CJ-Wright** [comment]
Yep

**Comment by CJ-Wright** [comment]
There was a discussion of having a callback which handles this to some extent

**Comment by CJ-Wright** [comment]
See https://github.com/NSLS-II/databroker/issues/253

**Comment by chiahaoliu** [comment]
@CJ-Wright Is this still an issue?

**Comment by CJ-Wright** [comment]
Yes, I imagine that this would be somewhere in `_end_beamtime` potentially with a kwarg to say that we want to export a databroker?
